monailabel/scribbles/utils.py

```diff
@@ -110,13 +110,18 @@ def make_iseg_unary(
 
 
 def make_histograms(image, scrib, scribbles_bg_label, scribbles_fg_label, bins=32):
+    # initialise prior distribution with a special case of Dirichlet distribution
+    # this helps with the case where only foreground scribbles are given,
+    # and vice versa for background scribbles
+    prior_values = [i / bins for i in range(bins)]
+
     # collect background voxels
-    values = image[scrib == scribbles_bg_label]
+    values = np.concatenate([image[scrib == scribbles_bg_label], prior_values], axis=0)
```

Comment:
As per the comment above rather than concatenating values it might be more intuitive to add a prior histogram of pseudo-count and the scribble-based histogram (with `density=False`) and then normalising the summed histogram.

Reply:
Thanks, I will update the code.